extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/common/KoofrClient.java

```diff
@@ -102,13 +103,9 @@ public boolean fileExists(String path) throws IOException, InvalidTokenException
       if (code == 404) {
         return false;
       }
-      throw new IOException(
-          "Got error code: "
-              + code
-              + " message: "
-              + response.message()
-              + " body: "
-              + response.body().string());
+      ResponseBody body = response.body();
```

Comment:
should we updat ethis method signature to clarify that it throws the more specific exception now?

Reply:
getResponse can also throw IOException, so it is probably better to leave it as is.